docs/source/data/data_catalog_yaml_examples.md

```diff
@@ -0,0 +1,395 @@
+# Data Catalog YAML examples
+
+This page contains a set of examples to help you structure your YAML configuration file in `conf/base/catalog.yml` or `conf/local/catalog.yml`.
+
+<!-- To do: fix anchors -->
+
+* [Load data from a local binary file using utf-8 encoding](#load-data-from-a-local-binary-file-using-utf-8-encoding)
```

Comment:
What about including a local table of contents, instead of hardcoding all the sections?

https://stackoverflow.com/a/24160594/554319

In MyST I guess it should be

````
```{contents} Table of Contents
:depth: 3
```
````

Reply:
I do like to use brute force rather than elegance. Thanks!